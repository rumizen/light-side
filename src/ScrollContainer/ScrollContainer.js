import React from 'react';
import "react-star-wars-crawl/lib/index.css";
import './ScrollContainer.scss';
import Crawl from "react-star-wars-crawl";

const ScrollContainer = ({ movie }) => {
  return (
    <Crawl
      className="scroll-container"
      title={movie.episode_id}
      subTitle={movie.title}
      text={movie.opening_crawl}
      height="20%"
    />
  );
}

export default ScrollContainer;