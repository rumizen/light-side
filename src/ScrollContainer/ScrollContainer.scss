.scroll-container {
  height: 200px;
  color: white;
  padding: 20px 215px;
}