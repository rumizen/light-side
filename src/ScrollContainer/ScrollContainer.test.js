import React from "react";
import { shallow, mount } from "enzyme";
import ScrollContainer from "./ScrollContainer";

describe("ScrollContainer", () => {
  it("should match the ScrollContainer snapshot", () => {
    const wrapper = shallow(
      <ScrollContainer
        movie="Return of the Jedi"
      />);

    expect(wrapper).toMatchSnapshot();
  });
});
