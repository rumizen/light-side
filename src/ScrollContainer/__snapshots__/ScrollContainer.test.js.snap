// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ScrollContainer should match the ScrollContainer snapshot 1`] = `ShallowWrapper {}`;
